src/sentry/celery.py

```diff
@@ -85,11 +86,20 @@ def celery_prefork_freeze_gc(**kwargs: object) -> None:
 class SentryTask(Task):
     Request = "sentry.celery:SentryRequest"
 
-    def delay(self, *args, **kwargs):
+    @classmethod
+    def _add_metadata(cls, kwargs: dict[str, Any]) -> None:
+        """
```

Comment:
hmm, should I do a `None` check here and also update the type hinting?

Reply:
Kwargs should always be a dict based on how this method is being called. A none check wouldn't hurt though.

Reply:
done, can't be too safe